src/database/models/highscores.py

```diff
@@ -105,3 +118,107 @@ class PlayerHiscoreData(Base):
     the_leviathan = Column(Integer, default=0)
     the_whisperer = Column(Integer, default=0)
     vardorvis = Column(Integer, default=0)
+
+
+# CREATE TABLE scraper_data (
+#   scraper_id BIGINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+#   created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+#   player_id SMALLINT UNSIGNED NOT NULL,
+#   record_date DATE AS (DATE(created_at)) STORED,
+#   UNIQUE KEY unique_player_per_day (player_id, record_date)
+# );
+class ScraperData(Base):
+    __tablename__ = "scraper_data"
+
+    scraper_id = Column(BigInteger, primary_key=True, autoincrement=True)
+    created_at = Column(DateTime, nullable=False, server_default=func.now())
+    player_id = Column(SmallInteger, nullable=False)
+    record_date = Column(Date, nullable=True, server_onupdate=func.current_date())
+    player_skills = relationship("PlayerSkills", back_populates="scraper_data")
+    __table_args__ = (
+        UniqueConstraint("player_id", "record_date", name="unique_player_per_day"),
+    )
+
+
+# CREATE TABLE skills (
+#   skill_id TINYINT UNSIGNED AUTO_INCREMENT PRIMARY KEY, # < 255
+#   skill_name VARCHAR(50) NOT NULL,
+#   UNIQUE KEY unique_skill_name (skill_name)
+# );
+class Skills(Base):
+    __tablename__ = "skills"
+
+    skill_id = Column(TinyInteger, primary_key=True, autoincrement=True)
+    skill_name = Column(String(50), nullable=False)
+
+    player_skills = relationship("PlayerSkills", back_populates="skill")

```

Comment:
please remove relationship, it sometimes has unexpected results on the sql query